xtask/src/ci.rs

```diff
@@ -289,16 +289,14 @@ impl CiTask {
         .map_err(Into::into)
     }
 
-    /// Lint ruma-common with clippy with the nightly version and wasm target.
+    /// Lint ruma-common and ruma-events with clippy with the nightly version and wasm target.
```

Comment:
I guess we can change this to:

```suggestion
    /// Lint ruma with clippy with the nightly version and wasm target.
```